app/scripts/lib/url.js

```diff
@@ -9,35 +9,54 @@ define(function (require, exports, module) {
 
   const _ = require('underscore');
 
-  function searchParams (str, allowedFields) {
-    const search = (typeof str === 'string' ? str : window.location.search).replace(/^\?/, '');
-    if (! search) {
-      return {};
-    }
+  module.exports = {
+    /**
```

Comment:
I moved this method into the object so it'd be more like the rest. I also removed the fallback of `window.location.search`, and added a `trim`